### ISSUE TITLE
Warning: Overrides cannot find parts of workshop_activity

### ISSUE THREAD

**Issue by esmeetewinkel** [IDEMSInternational/parenting-app-ui#2015]

**Comment by chrismclarke** [comment]
Would it be possible to create a minimal debug sheet that we can use for testing what desired behaviour might look like? I'm slightly wary as I think sometimes this warning does catch genuine issues (e.g. typos when trying to override a default), but it would be good to try reduce number of false-positives when the child content purposefully omitted

**Comment by esmeetewinkel** [comment]
> Would it be possible to create a minimal debug sheet

I created debug sheets and updated the issue description accordingly.

**Comment by chrismclarke** [comment]
Thanks @esmeetewinkel 
I started to take a very quick look although realised it will probably be easier to visit after #2040 as that contains some extra methods for adding tests to frontend code which would probably be good to also add for this case.

I assume this isn't super high priority (but nice to have for sure), so will try revisit after #2040 is in.
